apps/files/tests/test_tasks.py

```diff
@@ -42,33 +42,6 @@ def file(self, **file_values):
         file_.update(**file_values)
         return file_
 
-    def test_record_action(self):
-        file_ = self.file(builder_version='1234500')
-        tasks.start_upgrade([file_.id], sdk_version='1.2')
-        assert self.urllib2.urlopen.called
-        url, args = self.urllib2.urlopen.call_args[0]
-        args = dict(urlparse.parse_qsl(args))
-        self.assertDictEqual(args, {
-            'addon': str(3615),
-            'file_id': str(file_.id),
-            'priority': 'low',
-            'secret': settings.BUILDER_SECRET_KEY,
-            'package_key': file_.builder_version,
-            'uuid': args['uuid'],  # uuid is random so steal from args.
-            'pingback': absolutify(reverse('amo.builder-pingback')),
-            'sdk_version': '1.2',
-        })
-        eq_(url, settings.BUILDER_UPGRADE_URL)
-
-    def test_jetpack_with_older_harness_opt(self):
-        file_ = self.file(builder_version=None)  # old harness options
-        tasks.start_upgrade([file_.id], sdk_version='1.2')
-        url, args = self.urllib2.urlopen.call_args[0]
-        args = dict(urlparse.parse_qsl(args))
-        eq_(args['location'], file_.get_url_path('builder'))
-        assert 'package_key' not in args, (
-            'Unexpected keys: %s' % args.keys())
-
```

Comment:
All of the repackager code and tests should really be removed, but I can't think of a good reason to remove these two specifically.